### DISCUSSION TITLE
error on importing a redirected file

### DISCUSSION THREAD

**Discussion by BananaAcid** [FranckFreiburger/vue3-sfc-loader#80] [Q&A]
**Describe the bug**

I am importing a unpkg.com file, wich gets redirected to the reqal file. Fetch gets it, as in the examples, but die module ... shows an error:

**To Reproduce**
in a the first .vue component, `app.vue` showing a hello world, and having:
```js
import { hexToRgba } from "//unpkg.com/hex-and-rgba";
```
results in `https://unpkg.com/hex-and-rgba@1.4.2/index.js` -- the Chrome Network tab shows the redirected request, dumping right in the `getFiles()` function shows the content.

and the error:
```
Uncaught (in promise) ReferenceError: hexToRgba is not defined
    at Object.eval (eval at A (vue3-sfc-loader:75),
    at A (vue3-sfc-loader:75)
    at vue3-sfc-loader:75
    at async vue3-sfc-loader:75
    at async w (vue3-sfc-loader:75)
    at async j (vue3-sfc-loader:75)
```

**Expected behavior**
I believe, any redirect should work.

**Versions**
 - Browser (name & version): Chrome latest
 - vue3-sfc-loader: 0.8.0

**Comment by FranckFreiburger** [comment]
Hi,
By default, vue3-sfc-loader is not configured to support remote origin.
Please see https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/faq.md#from-a-different-origin-remote-component

FYI, default `option.pathResolve` only resolves local paths

**Reply by BananaAcid** [reply]
import with https://unpkg.com/hex-and-rgba@1.4.2/index.js works just fine, going with the demo https://codepen.io/franckfreiburger/project/editor/AqPyBr ... I probably misunderstood what this is project is about.
